release_docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,109 @@
+
+## Participants:
+- Product Manager — The individual responsible for the overall direction and development of a software product at The HDF Group.
+- Release Manager — The individual responsible for coordinating and overseeing the activities specific to releasing a software product at The HDF Group.
+- Software Developer — An individual or group responsible for performing maintenance on supported HDF software.
+- Software Tester — An individual responsible for identifying, implementing, and conducting tests of software code, modules, packages, executables, and/or release binaries.
+- Test Automation Team — A team at The HDF Group responsible for overseeing the the daily automated regression tests of software maintained by the company's Sustaining Engineering program.
+
+## Tasks:
+1. Plan Maintenance Release (Product Manager | Release Manager)
+
+2. Perform Software Maintenance (Software Developers | Product Manager | Release Manager)
+
+3. Prepare Release Notes (Release Manager)
+    - Audit the sufficiency of and update the release notes.
+
+4. Freeze Code (Release Manager | Test Automation Team)
+    - Transition from performing maintenance on software to preparing for its delivery.
+
+5. Update Interface Version (Release Manager | Product Manager)
+    - Verify interface additions, changes, and removals, and update the shared library interface version number.
+
+6. Prepare Release Branch (Release Manager)
+    - Get the release branch ready for pre-release testing and packaging.
+    - For all release preparation operations, the release coordinator will clone and push directly to canonical HDF5:
+        - `$ git clone ​https://github.com/HDFGroup/hdf5.git`
+
+7. Perform Release Testing (Test Automation Team | Release Manager | Project Leads)
+    - Verify that source and binary distributions of HDF5 are acceptable on all target operating environments.
+
+8. Finalize Release Notes (Release Manager)
+
+9. Package and Distribute Release (Release Manager)
+
+10. Conduct Release Retrospective (Release Manager)
+
+| Task Name     | Task Items    | Task Details | 
+| ------------- | ------------- | ---------- |
+| 3. Prepare Release Notes (Release Manager) | | |
+| | 1. Audit Release Changes (Release Manager) | Confirm that all non-trivial changes made to the source are reflected in the release notes. Verify the following: <br> [HDF5 Milestones Projects](https://github.com/HDFGroup/hdf5/milestones) <br> Each entry in [RELEASE.txt](https://github.com/HDFGroup/hdf5/blob/develop/release_docs/RELEASE.txt) traces to one or more resolved GH issues marked with FixVersion="X.Y.Z". <br> Each resolved GH milestone issue traces to an entry in [RELEASE.txt](https://github.com/HDFGroup/hdf5/blob/develop/release_docs/RELEASE.txt).<br>Each resolved GH milestone issue traces to one or more revisions to the HDF5 source. <br> Each resolved GH milestone issue traces to one or more pull requests. |
```

Comment:
Can we put these numbered substeps under the Task before the table and then just use the Task Details as a list?

Reply:
Sorry, can I get a bit more clarification on what you want?

Reply:
Decompose the table moving each row of a task and steps under the task # above the table
line 36 is replaced by :
 1. Audit Release Changes (Release Manager)
     - Confirm that all non-trivial changes made to the source are reflected in the release notes.
     - Verify the following:
           - [HDF5 Milestones Projects](https://github.com/HDFGroup/hdf5/milestones)
           - Each entry in [RELEASE.txt](https://github.com/HDFGroup/hdf5/blob/develop/release_docs/RELEASE.txt) traces to one or more resolved GH issues marked with FixVersion="X.Y.Z".
           -  ....
 3. Confirm and Update Known Issues (Release Manager or Software Maintainers)

Reply:
This has also been updated.